### DISCUSSION TITLE
Displaying navigation for page not included in nav

### DISCUSSION THREAD

**Discussion by jimlyas** [squidfunk/mkdocs-material#2986] [Q&A] [answered]
So i generated my javadoc to markdown and want to display it mkdocs, but when it open nested file, the navigation is disappear. Is it possible to keep displaying the navigation even in the page that not registered in `nav` variable in `mkdocs.yml`?

**Comment by squidfunk** [comment, marked as answer]
No, that's not possible. You need to register it in the `nav` entry:

> Navigation items may also include links to external sites. While titles are optional for internal links, they are required for external links. An external link may be a full URL or a relative URL. Any path which is not found in the files is assumed to be an external link. See the section about [Meta-Data] on how MkDocs determines the page title of a document.

https://www.mkdocs.org/user-guide/configuration/#nav

**Comment by hhslepicka** [comment]
@jimlyas what are you using to convert your javadoc into markdown? I am also building a documentation portal and I would like to inject my javadoc there.

@squidfunk sorry for bringing up an old discussion.

**Reply by jimlyas** [reply]
I'm sorry, I'm not really using javadoc. But I use Kdocs because the code is in Kotlin. For converting Kdoc into markdown you can use [Dokka](https://github.com/kotlin/dokka).

For answering my question, It's kinda hard adding all the pages to `mkdocs.yml` for all the documentation. So I use this [plugin](https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin) that scan the corresponding regex to automatically include it.
